frontend/src/app/game/status-display/status-display.component.ts

```diff
@@ -11,14 +11,18 @@ import {GameService} from '../game.service';
   styleUrls: ['./status-display.component.scss'],
 })
 export class StatusDisplayComponent {
+  lastState$ = this.gameService.gameState$.pipe(
+    filter(states => states.length > 0),
+    map(states => states[states.length - 1]),
```

Comment:
For reset view when game restarted could be better
`map(states => states?.length ? states[states.length - 1] : undefined),`

Next pipes must be edited to correctly process `undefined`.